fusedrug/data/tokenizer/modulartokenizer/multi_tokenizer.py

```diff
@@ -10,16 +10,23 @@
 import transformers
```

Comment:
please use a consistent name - modular_tokenizer or multi_tokenizer ,
and use it for all classes and filenames
(the file here is multi_tokenizer and the class name is modular tokenizer)

Reply:
What he said ^

Reply:
Done